### ISSUE TITLE
Having extra boarder in right and bottom sides / can not send options.Extend to vipsAffine

### ISSUE THREAD

**Issue by eslam-mahmoud** [h2non/bimg#308]
I created a PR for the proposed fix https://github.com/h2non/bimg/pull/307

**Orignal issue is** 
Having an extra border on the right and bottom side of images

**Root cause comes from**
https://libvips.github.io/libvips/API/current/libvips-resample.html#vips-resize
> When upsizing (scale > 1), the operation uses vips_affine() with a VipsInterpolate selected depending on kernel . It will use VipsInterpolateBicubic for VIPS_KERNEL_CUBIC and above. It adds a 0.5 pixel displacement to the input pixels to get center convention scaling.

**The fix**
Enable vipsAffine to use `Extend` option value and send it to lipvips
This will change the default from the one that libvips use which is `background` to the ones that bimg use which is  `C.VIPS_EXTEND_BLACK`
But because the lip add extra 1 or .5 pixel the background is considered black anyway so this will not affect anyone but will fix the bug of having a border on the right and bottom of some images

Creating 100x100 white jpg reproduced the issue if it is scaled even to 101x101 with these options 
```
bimg.Options{
		Width:         width,
		Height:        height,
		Embed:         false,
		NoProfile:     true,
		StripMetadata: true,
		Quality:       100,
	}
```
now adding this make it works
```
		Extend:        bimg.ExtendCopy
```

100x100 
![white](https://user-images.githubusercontent.com/1906758/68959361-7f90bb80-0793-11ea-9ce3-55582251cc15.jpg)

500x500
![whie-with-border](https://user-images.githubusercontent.com/1906758/68959371-83244280-0793-11ea-9b78-2897f966d5a9.jpg)

Note the more you scale the image up the darker and thicker the boarder gets

**Comment by Offspring2099** [comment]
+1 for this

**Comment by h2non** [comment]
Solved in `v1.1.0` by using `bimg.ExtendCopy` option. Closing.
